bin/sam_translate.py

```diff
@@ -5,9 +5,9 @@
 Known limitations: cannot transform CodeUri pointing at local directory.
 
 Usage:
-  sam-translate.py --template-file=sam-template.yaml [--verbose] [--output-template=<o>]
-  sam-translate.py package --template-file=sam-template.yaml --s3-bucket=my-bucket [--verbose] [--output-template=<o>]
-  sam-translate.py deploy --template-file=sam-template.yaml --s3-bucket=my-bucket --capabilities=CAPABILITY_NAMED_IAM --stack-name=my-stack [--verbose] [--output-template=<o>]
+  sam_translate.py --template-file=sam-template.yaml [--verbose] [--output-template=<o>] [--disable-api-configuration]
```

Comment:
Had to add this parameter here as well, otherwise docopt will generate Usage warning.

Reply:
What that makes no sense 🤔